specification/patient-care-aggregator-api.yaml

```diff
@@ -92,7 +91,7 @@ info:
     | ----------------- | ----------------------------------------------------------------------------- | ----------------------------- |
     | Sandbox           | Not available                                                                 | Not applicable                |
     | Integration test  | `https://int.api.service.nhs.uk/patient-care-aggregator-api/FHIR/R4/`         | NHS login - integration (AOS) |
-    | Production        | Not yet available                                                             | NHS login - Live (production) |
+    | Production        | `https://api.service.nhs.uk/patient-care-aggregator-api/FHIR/R4/`             | NHS login - Live (production) |
```

Comment:
The production URL is https://api.service.nhs.uk/patient-care-aggregator-api/aggregator/events. The change to /FHIR/R4 is on our backlog and will happen after we have gone live with the service as a fast follow

Reply:
Changed